website/docs/docs/introduction.md

```diff
@@ -1,6 +1,7 @@
 ---
 title: "What is dbt?"
 id: "introduction"
+description: "Welcome to dbt!"
```

Comment:
i think we can add more description to this page, which is heavily visited. 
```suggestion
description: "dbt is the industry standard for data transformation. Transform raw data into analysis-ready insights, and make data-driven decisions with confidence."
```

Reply:
This is hitting the top side of the character limit. We might want to consider brevity here
```suggestion
description: ""dbt is the industry standard for data transformation."
```

Reply:
Or
```suggestion
description: "dbt transforms raw data into analysis-ready insights and drives confident, data-informed decisions."
```